CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.6.0
+### Added
+* Service: The service now auto-scales the number of Azure Functions instances as needed [#238](https://github.com/microsoft/onefuzz/pull/238)
+* CLI/Service/Agent: Added the ability to set ensemble synchronization timers (including disabling ensemble all-together) [#229](https://github.com/microsoft/onefuzz/pull/229)
```

Comment:
For clarity, consider documenting this as "ensemble synchronization [interval/frequency]". Also, nit: want "altogether".